composer.json

```diff
@@ -10,5 +10,10 @@
     "require": {
         "php": ">=5.5",
         "zendframework/zendframework": "~2.5"
+    },
+    "autoload": {
+        "psr-4": {
+          "Application\\": "module/Application/src/Application"
```

Comment:
I'm :+1: to move the contents of `module/Application/src/Application` down to `module/Application/src`.